src/createServer.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const express = require('express');
+const cors = require('cors');
+const { router: usersRouter } = require('./routes/users.js');
+const { router: expensesRouter } = require('./routes/expenses.js');
+
+function createServer() {
+  const app = express();
+
+  app.use(cors());
+  app.use('/users', express.json(), usersRouter);
+  app.use('/expenses', express.json(), expensesRouter);
```

Comment:
It would be better to write this part in such a way

```suggestion
  const app = express();

  app.use(cors());
  app.use(express.json());
  app.use('/users', usersRouter);
  app.use('/expenses', expensesRouter);
```